Library/Formula/truncate.rb

```diff
@@ -0,0 +1,18 @@
+class Truncate < Formula
+  desc "truncates a file to a given size"
+  homepage "http://www.vanheusden.com/truncate"
+  url "https://github.com/flok99/truncate/archive/fc30a29dab789249f0c5f6863fdc5c8d4c9c3f69.zip"
```

Comment:
Ok it says it created tar-balls.

On Thu, Jan 14, 2016 at 10:57 AM, Dominyk Tiller <notifications@github.com>
wrote:

> In Library/Formula/truncate.rb
> <https://github.com/Homebrew/homebrew/pull/47883#discussion_r49706315>:
>
> > @@ -0,0 +1,18 @@
> > +class Truncate < Formula
> > +  desc "truncates a file to a given size"
> > +  homepage "http://www.vanheusden.com/truncate"
> > +  url "https://github.com/flok99/truncate/archive/fc30a29dab789249f0c5f6863fdc5c8d4c9c3f69.zip"
>
> Hey! Thanks for the help with this PR.
>
> I think you want a full tag rather than a lightweight one. Git doesn't
> treat the latter like actual releases and GitHub doesn't generate automatic
> tarballs for them. There's some relatively gentle documentation here
> <https://git-scm.com/book/en/v2/Git-Basics-Tagging> which you may find
> useful.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/47883/files#r49706315>.
>



-- 
www.vanheusden.com  www.slimwinnen.nl | www.smartwinning.info